geonode_mapstore_client/client/themes/geonode/less/_resources-grid.less

```
@@ -88,3 +88,42 @@
         max-width: 100%;
     }
 }
+
+.gn-resources-grid {
+    &.gn-panel {
+        height: 100%;
+
+        .gn-card-grid-container {
+            overflow: auto;
+        }
+
+        .gn-grid-container,
+        .gn-card-grid,
+        .gn-card-grid > div,
+        .gn-card-grid > div > div,
+        .gn-card-grid-container {
+            min-height: auto;
+            height: 100%;
+        }
+
+        .gn-resources-panel-wrapper {
+            position: absolute;
+        }
+
+        .gn-resources-filter, .gn-resource-detail {
+            max-width: 100%;
+        }
```

My proposal is to keep the max with of filter still at 400px as in the issue description

```suggestion
        .gn-resource-detail {
            max-width: 100%;
        }
        .gn-resources-filter {
            width: 100%;
            max-width: 400px;
        }
```